test/ci_concurrent_backups.js

```diff
@@ -18,7 +18,7 @@
 const fs = require('fs');
 const readline = require('readline');
 const u = require('./citestutils.js');
-const uuid = require('uuid/v4');
+const uuid = require('uuid').v4;
```

Comment:
Do we need to update the copyright?

Reply:
I think for a change that small, no, but it doesn't hurt either. Done in 8333d4c91e217509b0a7d2d1591c0f7b22cce038